src/components/TodoList.js

```diff
@@ -0,0 +1,25 @@
+import propTypes from "prop-types";
+import * as React from "react";
+import TodoListItem from "./TodoListItem";
+
+function TodoList(props) {
+console.log(props.todoList);
+TodoList.propTypes={
+  props: propTypes.string
+}
+  return (
+    <ul>
+
+      {props.todoList.map(function (item) {
```

Comment:
Here we have `props.todooList`, where we expect `todoList` to be an array, so in `TodoList.propTypes`, we should define `props.todoList` as `propTypes.array`.